go-controller/pkg/config/config.go

```diff
@@ -977,6 +978,11 @@ var K8sFlags = []cli.Flag{
 		Destination: &cliConfig.Kubernetes.PlatformType,
 		Value:       Kubernetes.PlatformType,
 	},
+	&cli.StringFlag{
+		Name:        "healthz-bind-address",
+		Usage:       "The IP address and port for the node proxy healthz server to serve on (set to '0.0.0.0' or '::' for listening in all interfaces and IP families). Disabled by default.",
```

Comment:
shouldn't this example include the port (and probably just use the default that kube-proxy does) like set to '0.0.0.0:10256' or '::10256' for listening in all interfaces and IP families using kube-proxy default healthz port 10256.

Reply:
@trozet I copied what kube-proxy uses for an argument, but that sucks, so I'll change it.

Reply:
Fixed.